src/main/java/org/folio/service/CirculationSettingsService.java

```diff
@@ -54,17 +60,53 @@ public Future<Response> findById(String circulationSettingsId) {
       GetCirculationSettingsStorageCirculationSettingsByCirculationSettingsIdResponse.class);
   }
 
-  public Future<Response> update(String circulationSettingsId, CirculationSetting circulationSetting) {
-    return PgUtil.put(CIRCULATION_SETTINGS_TABLE, circulationSetting, circulationSettingsId, okapiHeaders, vertxContext,
+  public Future<Response> update(String circulationSettingsId,
+    CirculationSetting circulationSetting) {
+
+    return PgUtil.put(CIRCULATION_SETTINGS_TABLE, circulationSetting, circulationSettingsId,
+        okapiHeaders, vertxContext,
         PutCirculationSettingsStorageCirculationSettingsByCirculationSettingsIdResponse.class)
       .compose(eventPublisher.publishUpdated(circulationSetting));
   }
 
   public Future<Response> delete(String circulationSettingsId) {
-    return repository.getById(circulationSettingsId).compose (
-      circulationSetting -> PgUtil.deleteById(CIRCULATION_SETTINGS_TABLE, circulationSettingsId, okapiHeaders, vertxContext,
-        DeleteCirculationSettingsStorageCirculationSettingsByCirculationSettingsIdResponse.class)
-      .compose(eventPublisher.publishRemoved(circulationSetting))
+    return repository.getById(circulationSettingsId)
+      .compose(circulationSetting -> PgUtil.deleteById(CIRCULATION_SETTINGS_TABLE,
+          circulationSettingsId, okapiHeaders, vertxContext,
+          DeleteCirculationSettingsStorageCirculationSettingsByCirculationSettingsIdResponse.class)
+        .compose(eventPublisher.publishRemoved(circulationSetting))
     );
   }
+
+  private Future<CirculationSetting> updateSettingsValue(CirculationSetting circulationSetting,
+    Throwable throwable) {
+
+    if (!isDuplicate(throwable.getMessage())) {
+      log.info("updateSettingsValue:: setting with name: {} already exists.",
```

Comment:
If we enter this `if`, it means that the error is not a unique constraint violation